README.md

```diff
@@ -97,7 +97,7 @@ and uninstalled through
 
     $ sudo make uninstall.comp
```

Comment:
You mean, `PREFIX=/usr` only for the `.comp` targets?

> We can safely assume very few would be interested in reading their shell's manual to install scripts that come with instructions to install.

That could be true to some extent. Which is pretty sad.